src/main/java/thePackmaster/cards/lockonpack/TunnelVision.java

```diff
@@ -23,6 +25,10 @@ public void upp() {
 
     @Override
     public void use(AbstractPlayer p, AbstractMonster m) {
+        if (magicNumber > 0)
+        {
+            for (AbstractMonster mon : Wiz.getEnemies()) addToBot(new ApplyPowerAction(mon, p, new LockOnPower(mon, magicNumber)));
```

Comment:
We have an action to apply powers to all enemies. It's a lot faster than doing ti one by one, please use that